docs/mac-broker-integration.md

```diff
@@ -0,0 +1,20 @@
+# macOS broker integration
+
+This doc indicates how to use `msal` to communicate with the `broker on macOS` (Company Portal), which provides more secure authentication and single sign on experiences.
+
+### Manually testing/integration steps
+1. Install CP (Company Portal) on the mac device, login an acount in CP and finish the MDM process.
+2. Install msal: `pip install msal==1.27.0b2`
+3. Install pymsalruntime: `pip install -i https://test.pypi.org/simple/ pymsalruntime==0.17.0`
+
+### Verification msal and pymsalruntime works fine
+
+Please refer to [broker-test.py](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/dev/tests/broker-test.py) as a sample. Notice, we use `enable_broker_on_mac=True` on macOS.
+```
+pca = msal.PublicClientApplication(
+    _AZURE_CLI,
+    authority="https://login.microsoftonline.com/organizations",
+    enable_broker_on_mac=True)
+```
```

Comment:
Since we already maintain `broker-test.py`, there is no point to duplicate its code snippet here.

```suggestion
Please refer to [broker-test.py](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/dev/tests/broker-test.py) as a sample.
```